core/src/main/scala/org/apache/spark/scheduler/TaskInfo.scala

```diff
@@ -74,6 +81,10 @@ class TaskInfo(
     gettingResultTime = time
   }
 
+  private[spark] def markKilledAttempt: Unit = {
```

Comment:
I think I suggested you to add parenthesis in this method signature `def markKilledByOtherAttempt()`, can you please change it?

> It is better to change the method signature to def markKilledByOtherAttempt(): Unit = {, since this method has side affect, it is better to add parenthesis.

Reply:
Sorry for the missing, I change it right now.